frontend/src/routes.js

```diff
@@ -1,3 +1,5 @@
+const base = 'https://speedy-anthem-217710.an.r.appspot.com';
```

Comment:
why is this needed ?

Reply:
For production, it is not needed. During development, the ReactJS app and Spring app can be on different domains - having the base URL for APIs at one place comes in handy then.